<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProperty</class>
 <widget class="QWidget" name="FormProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>属性</string>
  </property>
  <property name="windowIcon">
   <iconset resource="HPGCToolbox.qrc">
    <normaloff>:/toolset</normaloff>:/toolset</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>一般</string>
    </attribute>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>365</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>365</width>
       <height>121</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>标签</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>365</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>算法包</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>描述</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenFile">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/openfolder</normaloff>:/openfolder</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabHelp">
    <attribute name="title">
     <string>帮助</string>
    </attribute>
    <widget class="QTextEdit" name="textHelp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>365</width>
       <height>325</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>383</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>383</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="HPGCToolbox.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>FormProperty</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
